royale_api_website_scraper.py

```diff
@@ -0,0 +1,69 @@
+import re
+import socket
+from collections import OrderedDict
+
+import requests
+from bs4 import BeautifulSoup
+from requests.packages import urllib3
+
+
+# To hide warning about:
+# InsecureRequestWarning: Unverified HTTPS request is being made to host 'royaleapi.com'. Adding certificate verification
+# is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+class RoyaleApiWebsiteScraper:
+    def __init__(self):
+        self.soup = BeautifulSoup()
+
+    def get_html_soup(self, url):
+        # Followed approach here to avoid Cloudflare captcha / cookie / JS check
+        # "...can now bypass the cloudflare block using requests
+        # as long as we connect directly to the host IP rather than the domain name..."
+        # https://stackoverflow.com/questions/62684468/pythons-requests-triggers-cloudflares-security-while-urllib-does-not
+
+        answers = socket.getaddrinfo('royaleapi.com', 443)
+        (family, type, proto, canonname, (address, port, _, _)) = answers[0]
```

Comment:
what ise the purpose of the two lines (with answers variable)?

Reply:
Guess it's not needed lol. Have now removed